source/retryable-writes/retryable-writes.rst

```diff
@@ -408,17 +422,18 @@ Q & A
 Why are write operations only retried once?
 -------------------------------------------
 
-The spec concerns itself with retrying write operations that fail to return a
-response due to a network error, which may be classified as either a transient
-error (e.g. dropped connection, replica set failover) or persistent outage. In
-the case of a transient error, the driver will mark the server as “unknown” per
-the `SDAM`_ spec. A subsequent retry attempt will allow the driver to rediscover
-the primary within the designated server selection timeout period (30 seconds by
-default). If server selection times out during this retry attempt, we can
-reasonably assume that there is a persistent outage. In the case of a persistent
-outage, multiple retry attempts are fruitless and would waste time. See
-`How To Write Resilient MongoDB Applications`_ for additional discussion on this
-strategy.
+The spec concerns itself with retrying write operations that encounter a
+retryable error (i.e. no response due to network error or a response indicating
+that the node is no longer a primary). A retryable error may be classified as
+either a transient error (e.g. dropped connection, replica set failover) or
+persistent outage. In the case of a transient error, the driver will mark the
+server as "unknown" per the `SDAM`_ spec. A subsequent retry attempt will allow
+the driver to rediscover the primary within the designated server selection
+timeout period (30 seconds by default). If server selection times out during
+this retry attempt, we can reasonably assume that there is a persistent outage.
+In the case of a persistent outage, multiple retry attempts are fruitless and
+would waste time. See `How To Write Resilient MongoDB Applications`_ for
+additional discussion on this strategy.
```

Comment:
Most of this paragraph is unchanged. I edited the first two sentences and then reapplied word wrap.